package/kernel/linux/modules/hwmon.mk

```diff
@@ -97,7 +97,7 @@ define KernelPackage/hwmon-nct6775
   TITLE:=NCT6106D/6775F/6776F/6779D/6791D/6792D/6793D and compatibles monitoring support
   KCONFIG:=CONFIG_SENSORS_NCT6775
   FILES:=$(LINUX_DIR)/drivers/hwmon/nct6775.ko
-  AUTOLOAD:=$(call AutoProbe,nct6775)
+  AUTOLOAD:=$(call AutoLoad,51,nct6775)
```

Comment:
@pprindeville got back to me and confirmed that using AutoProbe with patched w83627ehf works for him on hardware with NCT6775. Hence I merged that version.